tests/manage/test_pvc_delete_verify_size_is_returned_to_backendpool.py

```diff
@@ -0,0 +1,274 @@
+"""
+A test case to verify after deleting pvc whether
+size is returned to backend pool
+"""
+import logging
+import pytest
+
+import ocs.defaults as defaults
+from ocsci.testlib import tier1, ManageTest
+from utility.utils import run_cmd
+from utility import templating
+from ocs.ocp import get_ceph_tools_pod
+from ocs.utils import create_oc_resource
+from ocs import ocp, pod
+
+log = logging.getLogger(__name__)
+_templating = templating.Templating()
+
+# Project name
+PROJECT_NAME = "ocs-372"
+
+PRJ = ocp.OCP(kind='Project')
+
+# yaml path
+TEMPLATES_DIR = "/tmp/"
+
+template_data = {}
+
+# Service
+OCP = ocp.OCP(
+    kind='Service', namespace=defaults.ROOK_CLUSTER_NAMESPACE
+)
+
+# CephBlockPool
+CBP = ocp.OCP(
+    kind='CephBlockPool', namespace=defaults.ROOK_CLUSTER_NAMESPACE
+)
+
+# Storage class
+SC = ocp.OCP(
+    kind='StorageClass', namespace=defaults.ROOK_CLUSTER_NAMESPACE
+)
+
+# Secret
+SECRET = ocp.OCP(
+    kind='Secret', namespace="default"
+)
+
+# PVC
+PVC = ocp.OCP(
+    kind='PersistentVolumeClaim', namespace=PROJECT_NAME
+)
+
+# Pod
+POD = ocp.OCP(
+    kind='Pod', namespace=PROJECT_NAME
+)
+
+
+def create_cephblock_pool(pool_name):
+
+    """
+    Create a cephblockpool
+    """
+
+    template_data['rbd_pool'] = pool_name
+    create_oc_resource(
+        'CephBlockPool.yaml', TEMPLATES_DIR, _templating,
+        template_data, template_dir='CSI/rbd'
+    )
+
+    # Validate cephblock created on oc
+    assert CBP.get(f'{pool_name}')
+
+    # Validate cephblock created on ceph
+    _rc = False
+    pools = ocp.exec_ceph_cmd(ceph_cmd="ceph osd lspools")
+    for pool in pools:
```

Comment:
I'm not sure I understand the login here.
let's say you have 3 pools:
1) in case the pool_name is equal to pool['poolname'] --> rc = True
2) the next one is False
3) the 3rd is True again 
assert True! but you had 1 which is not equal

Please explain what you are trying to achieve here

Reply:
Here I need to verify the pool which created, is there in ceph backend also.

for e.g: "rbd-pool" was created and need to check whether "rbd-pool" exists in pool['poolname'], if yes _rc=True otherwise false